gobblin-runtime/src/main/java/gobblin/runtime/instance/StandardGobblinInstanceDriver.java

```diff
@@ -417,6 +418,11 @@ public boolean isInstrumentationEnabled() {
 
     public List<GobblinInstancePluginFactory> getPlugins() {
       List<GobblinInstancePluginFactory> res = new ArrayList<>(getDefaultPlugins());
+      // By default email notification plugin is added.
```

Comment:
I would prefer you use `PLUGINS_FULL_KEY`, which allows specifying arbitrary plugins in the system configuration.